.env

```diff
@@ -4,3 +4,4 @@ NEXT_PUBLIC_APP_AUTHOR_NAME=$npm_package_author_name
 
 NEXT_PUBLIC_DATABASE_URL=
 DATABASE_API_TOKEN=
+DATABASE_MEDIA_HOST=localhost
```

Comment:
to potrzebne? nie powinno być w `env.local`

Reply:
NextJS wymaga niepustego stringa do poprawnego budowania, przekazuję `localhost` jako bezpieczny fallback do builda bez lokalnej konfiguracji.